modules/updating-node-pools-for-hcp.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assembly:
+//
+// * updates/updating-hosted-control-planes.adoc
+
+:_content-type: PROCEDURE
+[id="updating-node-pools-for-hcp_{context}"]
+= Updating node pools for hosted control planes
+
+On hosted control planes, you update your version of {product-title} by updating the node pools. You must update node pools to the current version of the hosted control plane.
+
+//.Prerequisites
+
+//Does this process require any prerequisites?
+
+.Procedure
+
+. To start the process to update to a new version of {product-title}, change the `spec.release.image` value of the node pool.
+
+. To create a `MachineConfig` object inside of a config map in the management cluster, enter the following information:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: ${CONFIGMAP_NAME}
+  namespace: clusters
+data:
+  config: |
+    apiVersion: machineconfiguration.openshift.io/v1
+    kind: MachineConfig
+    metadata:
+      labels:
+        machineconfiguration.openshift.io/role: worker
+      name: ${MACHINECONFIG_NAME}
+    spec:
+      config:
+        ignition:
+          version: 3.2.0
+        storage:
+          files:
+          - contents:
+              source: data:...
+            mode: 420
+            overwrite: true
+            path: ${PATH}
+----
+
+. After you apply that object to the config map, you can specify the config map to the node pool as follows:
++
+[source,yaml]
+----
+spec:
+  config:
+    - name: ${CONFIGMAP_NAME}
+----
+
+
+//.Verification
+
+// How can users verify that the update occurred successfully?
```

Comment:
There's information on the status and conditions of each NodePool, e.g:
something in the line of "You can  verify the new version was rolled out succesfully by checking .status.version and the status conditions"